LiskHQ/lisk-hub
# Implement BTC transaction detail page - Closes #1929

### What issue have I solved?

#1929 

### How have I implemented/fixed it?

I implemented the page according to the design, except for:
- the title is capitalized ("Transfer Transaction") to be consistent with other titles
- amount value is 18px to be consistent with style guide and other pages (agreed by Dominic: https://lightcurve.slack.com/archives/CGBKV6GCA/p1557393358003900 )
- Tooltip for "Confirmations" is not implemented because we had only LSK-specific copy ([see row 14](https://docs.google.com/spreadsheets/d/1X-9Zx3M80bhU5OsuIyERex_UjGNo03DjPwQh8jPbKCs/edit#gid=0)), waiting for a copy that works also for BTC.

### How has this been tested?


1. Go to  https://jenkins.lisk.io/test/lisk-hub/PR-1992/
2. log it to testnet
2. Set `localStorage.setItem('btc', true);`
1. Go to https://jenkins.lisk.io/test/lisk-hub/PR-1992/#/explorer/transactions/553787aa91a55e0385beb81a0a549f02cb3be8492b452fe8b09c1729165a8362
3. see there a BTC transaction details.

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
